### ISSUE TITLE
Typo in AEDT syslib materials.amat

### ISSUE THREAD

**Issue by SMoraisAnsys** [ansys/pyedb#364]
There is a typo in the syslib `Materials.amat` that is bundled with AEDT.
Instead of having `thermal_expansion_coefficient` the key is `thermal_expansion_coeffcient` (lacking an "i") ....

**Comment by SMoraisAnsys** [comment]
@ring630 That's one of the point I mentioned during our talk about the Material refactoring

**Comment by ring630** [comment]
@SMoraisAnsys I filed a defect

**Comment by SMoraisAnsys** [comment]
> @SMoraisAnsys the DEV decides not to fix it. `These are not user-facing files. As long as the I/O works for this string and corresponds to the correct material property, it's okay.`

Yeah but the thing is that the string does not correspond. I'd prefer to avoid adding custom workarounds to fix issues coming from the AEDT installation... I'll speak about this with @MaxJPRey

**Comment by ring630** [comment]
@SMoraisAnsys @MaxJPRey 
BUG 1039361

**Comment by gkorompi** [comment]
development decided not to fix, as it the typo is not user-facing, and they dont want to change legacy code.
